src/intl/Intl.h

```diff
@@ -101,6 +103,24 @@ class Intl {
         }                                                                                                    \
         return std::make_pair(status, output);                                                               \
     })()
+
+#define INTL_ICU_STRING_BUFFER_OPERATION_COMPLEX(icuFnName, extra, ...)                                             \
```

Comment:
Why did you make it as macro consist of a lambda with capture, not a member method?

Reply:
I want to declare as member static-template-function.
but there is a problem.
with runtime ICU binder, compiler cannot deduce type of ICU functions.
that's why I declare as macro